addons/test_mass_mailing/tests/test_mass_mailing.py

```diff
@@ -311,3 +311,17 @@ def test_mass_mail_on_res_partner(self):
         # if user is opt_out on One list but not on another, send the mail anyway
         self.assertEqual(self.mass_mailing.ignored, 1,
                          'Blacklisted ignored email number incorrect, should be equals to 1')
+
+
+class TestMassMailingControllers(HttpCase):
+    def test_tracking_url_sucess(self):
```

Comment:
Can be moved in mass_mailing directly as it is not really linked to that test module, only some generic checks.